client/src/RatingsReviews/ReviewTile.jsx

```diff
@@ -0,0 +1,236 @@
+import React from 'react';
+import RRModal from './PhotoModal.jsx';
+import dummyReviews from './dummyReviews';
+
+// TODO: Fix technical debt. A11y issues, general formatting.
+// Tile needs to be a fixed size.
+// Update helpfulness state.
+// Update for use with real data.
+// Write some tests.
+// Clean up CSS
+
+class ReviewTile extends React.Component {
+  constructor(props) {
+    super(props);
+    // Single test review testing block.
+    let [testReview, testReview2] = dummyReviews.results;
+    // Calculate star rating.
+    if (testReview.rating > 0) {
+      testReview.starRating = parseFloat((Math.round(testReview.rating * 4) / 4).toFixed(2));
+    }
+    this.state = {
+      ...testReview,
+      showModal: false,
+      modalURL: '',
+      helpfulClicked: false,
+    };
+    this.state.body =
```

Comment:
Does this work? I thought we had to use this.setState() in order to change state?